### DISCUSSION TITLE
Docker

### DISCUSSION THREAD

**Discussion by ziggysdomain** [GyulyVGC/sniffnet#600] [Ideas]
Any chance you might be able to create a docker version of the app? Thanks

**Comment by GyulyVGC** [comment]
Hi @ziggysdomain thanks for your interest in Sniffnet.

This was already requested in issue #69.
We also have a PR open for this (#455) but we faced some problem to create the image because of the GUI library Sniffnet is using.

Follow that issue and PR for future updates.
